### DISCUSSION TITLE
Application Status in not getting updated on dashboard

### DISCUSSION THREAD

**Discussion by pranavcracker** [argoproj/argo-cd#20764] [Q&A]
I am facing an issue on argocd that, in actual the apps are getting installed on my kubernetes cluster but on the argocd dashboard those are showing as Missing.
Even if I try to sync them, getting error as "runtime error: invalid memory address or nil pointer dereference"
Any suggestions/solution on this situation?

**Comment by andrii-korotkov-verkada** [comment]
I suggest to open a bug issue with providing more details, like screenshots, logs, argocd version.
